ws/session.nim

```diff
@@ -68,7 +68,7 @@ proc send*(
   let maxSize = ws.frameSize
   var i = 0
   while ws.readyState notin {ReadyState.Closing}:
-    let len = min(data.len, (maxSize + i))
+    let len = min(data.len - i, maxSize)
```

Comment:
This seems like the actual bug for multiple frames.

Reply:
fixed